compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -1192,29 +1223,17 @@ def cmd_start_endpoint(
                     f"\n  (pid: {pid}, user: {uname}, {ep_name})"
                 )
 
-            os.setsid()
-            exit_code += 1
-
-            umask = 0o077  # Let child process set less restrictive, if desired
-            log.debug("Setting process umask for %s to 0o%04o (%s)", pid, umask, uname)
-            os.umask(umask)
-            exit_code += 1
-
             log.debug("Changing directory to '%s'", wd)
+            env["PWD"] = wd
             os.chdir(wd)
             exit_code += 1
 
-            os.environ["PWD"] = wd
-            os.environ["CWD"] = wd
-            env["PWD"] = wd
-            env["CWD"] = wd
-
             # in case "something gets stuck," let cmdline show it
             args_title = " ".join(proc_args)
             startup_proc_title = f"Endpoint starting up for {uname} [{args_title}]"
             setproctitle.setproctitle(startup_proc_title)
 
-            gc_dir: pathlib.Path = GC.sdk.compute_dir.ensure_compute_dir()
+            gc_dir: pathlib.Path = ensure_compute_dir()
             ep_dir = gc_dir / ep_name
             ep_dir.mkdir(mode=0o700, parents=True, exist_ok=True)
             ep_log = ep_dir / "endpoint.log"
```

Comment:
Not necessarily for this PR, but we should consider moving this code from the `EndpointManager` to the start UEP path.